test/JDBC/input/operator_binary_whitespace-vu-verify.sql

```diff
@@ -0,0 +1,139 @@
+-- bit AND/OR/XOR
+select 99&-5
+go
+select 99&+5
+go
+select 99&~5
+go
+declare @v int = 99 set @v = 123&@v select @v
+go
+
+select 88|-34
+go
+select 88|+34
+go
+select 88|~34
+go
+declare @v int = 88 set @v = 123|@v select @v
+go
+
+select 77^-34
+go
+select 77^+34
+go
+select 77^~34
+go
+declare @v int = 77 set @v = 123^@v select @v
+go
+declare @v int = 10&+2 set @v = @v|-9 select @v
+go
+select * from v1_bitop
+go
+execute p1_bitop
+go
+print 99&+2 
+go
+print 99&~1
+go
+
+
+-- bit NOT
+select ~+2
+go
+select +~+2
+go
+select -~+2
+go
+select ~-2
+go
+select +~-2
+go
+select -~-2
+go
+select +~2
+go
+select -~2
+go
+select ~~2
+go
+select ~~~~~2
+go
+select ~~~~~+2
+go
+select +~~~~~+2
+go
+select -~~~~~+2
+go
+select ~~~~~+2
+go
+select +~~~~~-2
+go
+select -~~~~~-2
+go
+select ~@@max_precision
+go
+declare @v int = 2 set @v = ~@@max_precision select @v
+go
+declare @v int = 2 set @v = ~@v select @v
+go
+select * from v1_bitop_not
+go
+execute p1_bitop_not 
+go
+print ~+10
+go
+declare @v int = 10 print ~@v
+go
+
+-- modulo
+select 10%-3
+go
+select 10%+3
+go
+select 100%@@max_precision
+go
+declare @v int select 99%@v
+go
+select * from v1_modulo_op
+go
+exec p1_modulo_op
+go
+declare @v int =10%+3 select @v
+go
+declare @v int =10%+~3 select @v
+go
+declare @v int =10%~+3 select @v
+go
+declare @v int = 10 set @v = 3%@v select @v

```

Comment:
After declare @v, would you add select @v so we'll be able to test and verify the output

Reply:
`select @v` is at the end of each line